walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -24,33 +24,28 @@
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    // TODO: Maybe use binary tree or another data structure for availableTimes 
-    // because branches split in two?
-    // TODO: Combine the repetition of calling getAvailableTimes() below so it
-    // doesn't need to repeat without optional attendees.
     Collection<TimeRange> availableTimes = new ArrayList<>();
+    Collection<TimeRange> availableTimesWithoutOptional;
     Collection<String> attendees = request.getAttendees();
```

Comment:
What this is specifically?  List vs. Set?  Thoughts on if one is more appropriate than the other for what you're building here?